### PR TITLE
more link fixes for npm run docs

### PR DESCRIPTION
**Pull request by trackleft**

### CODE REVIEW

**Review by joeparsons — APPROVED** [review]

File: site/content/docs/2.0/getting-started/build-tools.md [hunk 1]
```diff
@@ -84,16 +84,16 @@ Twitter Bootstrap provides a configuration file for called `config.yml` for the
 - `AZ_SITE_HOST` Name or IP address at which to serve the documentation site
 - `AZ_VERSION` Full current Arizona Bootstrap version number
 
-{{< ourname >}} uses [npm scripts](https://docs.npmjs.com/misc/scripts) for its build system. Our [package.json]({{ site.repo }}/blob/v{{ site.current_version }}/package.json) includes convenient methods for working with the framework, including compiling code, running tests, and more.
+{{< ourname >}} uses [npm scripts](https://docs.npmjs.com/misc/scripts) for its build system. Our [package.json]({{< param "repo" >}}/blob/v{{ site.current_version }}/package.json) includes convenient methods for working with the framework, including compiling code, running tests, and more.
```
Comment (trackleft) [inline site/content/docs/2.0/getting-started/build-tools.md:87]:
```suggestion
{{< ourname >}} uses [npm scripts](https://docs.npmjs.com/misc/scripts) for its build system. Our [package.json]({{< param "repo" >}}/blob/v{{< param "current_version" >}}/package.json) includes convenient methods for working with the framework, including compiling code, running tests, and more.
```

File: site/content/docs/2.0/getting-started/build-tools.md [hunk 2]
```diff
@@ -84,16 +84,16 @@ Twitter Bootstrap provides a configuration file for called `config.yml` for the
...
 
 To use our build system and run our documentation locally, you'll need a copy of {{< ourname >}}'s source files and Node. Follow these steps and you should be ready to rock:
 
 1. [Download and install Node.js](https://nodejs.org/en/download/), which we use to manage our dependencies.
-2. Navigate to the root `/arizona-bootstrap` directory and run `npm install` to install our local dependencies listed in [package.json]({{ site.repo }}/blob/v{{ site.current_version }}/package.json).
+2. Navigate to the root `/arizona-bootstrap` directory and run `npm install` to install our local dependencies listed in [package.json]({{< param "repo" >}}/blob/v{{ site.current_version }}/package.json).
```
Comment (trackleft) [inline site/content/docs/2.0/getting-started/build-tools.md:92]:
```suggestion
2. Navigate to the root `/arizona-bootstrap` directory and run `npm install` to install our local dependencies listed in [package.json]({{< param "repo" >}}/blob/v{{< param "current_version" >}}/package.json).
```

File: site/content/docs/2.0/getting-started/build-tools.md [hunk 3]
```diff
@@ -84,16 +84,16 @@ Twitter Bootstrap provides a configuration file for called `config.yml` for the
...
 
 ## Using npm Scripts
 
-Our [package.json]({{ site.repo }}/blob/v{{ site.current_version }}/package.json) includes the following commands and tasks:
+Our [package.json]({{< param "repo" >}}/blob/v{{ site.current_version }}/package.json) includes the following commands and tasks:
```
Comment (trackleft) [inline site/content/docs/2.0/getting-started/build-tools.md:96]:
```suggestion
Our [package.json]({{< param "repo" >}}/blob/v{{< param "current_version" >}}/package.json) includes the following commands and tasks:
```

File: site/content/docs/2.0/getting-started/build-tools.md [hunk 4]
```diff
@@ -107,7 +107,7 @@ Run `npm run` to see all the npm scripts.
 
 {{< ourname >}} uses [Autoprefixer][autoprefixer] (included in our build process) to automatically add vendor prefixes to some CSS properties at build time. Doing so saves us time and code by allowing us to write key parts of our CSS a single time while eliminating the need for vendor mixins like those found in v3.
 
-We maintain the list of browsers supported through Autoprefixer in a separate file within our GitHub repository. See [.browserslistrc]({{ site.repo }}/raw/v{{ site.current_version }}/.browserslistrc) for details.
+We maintain the list of browsers supported through Autoprefixer in a separate file within our GitHub repository. See [.browserslistrc]({{< param "repo" >}}/raw/v{{ site.current_version }}/.browserslistrc) for details.
```
Comment (trackleft) [inline site/content/docs/2.0/getting-started/build-tools.md:110]:
```suggestion
We maintain the list of browsers supported through Autoprefixer in a separate file within our GitHub repository. See [.browserslistrc]({{< param "repo" >}}/raw/v{{< param "current_version" >}}/.browserslistrc) for details.
```

File: site/content/docs/2.0/getting-started/javascript.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ If you use a bundler (Webpack, Rollup...), you can use `/js/dist/*.js` files whi
 
 ## Dependencies
 
-Some plugins and CSS components depend on other plugins. If you include plugins individually, make sure to check for these dependencies in the docs. Also note that **all plugins depend on jQuery** (this means jQuery must be included **before** the plugin files). [Consult our `package.json`]({{ site.repo }}/blob/v{{ site.current_version }}/package.json) to see which versions of jQuery are supported.
+Some plugins and CSS components depend on other plugins. If you include plugins individually, make sure to check for these dependencies in the docs. Also note that **all plugins depend on jQuery** (this means jQuery must be included **before** the plugin files). [Consult our `package.json`]({{< param "repo" >}}/blob/v{{ site.current_version }}/package.json) to see which versions of jQuery are supported.
```
Comment (trackleft) [inline site/content/docs/2.0/getting-started/javascript.md:17]:
```suggestion
Some plugins and CSS components depend on other plugins. If you include plugins individually, make sure to check for these dependencies in the docs. Also note that **all plugins depend on jQuery** (this means jQuery must be included **before** the plugin files). [Consult our `package.json`]({{< param "repo" >}}/blob/v{{< param "current_version" >}}/package.json) to see which versions of jQuery are supported.
```

**Review by mmunro-ltrr — APPROVED** [review]
Can we just merge this into feature/219 right away? (obvious fixes for oversights there).

**Pull request merged**